### ISSUE TITLE
Add a rules 'import' and 'export' option to the APT package.

### ISSUE THREAD

**Issue by ElleLeonne** [evilsocket/opensnitch#1123]
### Summary:
I noticed something odd when using the APT version:
There's no 'export' or 'import' ruleset options.
This seems like such a simple addition, given you already support parsing lists, may as well add the option to import them too.

I suppose a follow up question I have is, where are these stored, so that I can manually create backups? It's troublesome to recreate the ruleset every install.

I'm unsure if the standalone version has these features but, if so, even better, the work is already done.

**Comment by ElleLeonne** [comment]
Okay, so I had searched the issues, and the wiki, before posting this, but ultimately found what I need here:

https://github.com/evilsocket/opensnitch/wiki/Configurations

tldr; they're stored under /etc/opensnitchd/rules/*.json
